scripts/addons/io_scene_gltf2/gltf2_generate.py

```diff
@@ -958,37 +958,19 @@ def generate_lights_cmn(operator,
         else:
             continue
 
-        if blender_light.type == 'POINT' or blender_light.type == 'SPOT':
-
-            positional = {}
-
-            if blender_light.falloff_type == 'CONSTANT':
-                positional['constantAttenuation'] = 1.0
-            elif blender_light.falloff_type == 'INVERSE_LINEAR':
-                positional['linearAttenuation'] = 1.0 / blender_light.distance
-            elif blender_light.falloff_type == 'INVERSE_SQUARE':
-                positional['quadraticAttenuation'] = 1.0 / blender_light.distance
-            elif blender_light.falloff_type == 'INVERSE_COEFFICIENTS':
-                positional['constantAttenuation'] = blender_light.constant_coefficient * 1.0
-                positional['linearAttenuation'] = blender_light.linear_coefficient * 1.0 / blender_light.distance
-                positional['quadraticAttenuation'] = blender_light.quadratic_coefficient * 1.0 / blender_light.distance
-            else:
-                continue
-
-            light['positional'] = positional
+        if blender_light.type == 'SPOT':
+            spot = {}
 
-            if blender_light.type == 'SPOT':
-                spot = {}
+            angle = blender_light.spot_size * 0.5
 
-                angle = blender_light.spot_size * 0.5
+            spot['outerConeAngle'] = angle
+            spot['innerConeAngle'] = angle - angle * blender_light.spot_blend
 
-                spot['outerAngle'] = angle
-                spot['innerAngle'] = angle - angle * blender_light.spot_blend
+            light['spot'] = spot
 
-                positional['spot'] = spot
-
-        light['color'] = [blender_light.color[0] * blender_light.energy, blender_light.color[1] * blender_light.energy,
-                          blender_light.color[2] * blender_light.energy]
+        light['color'] = [blender_light.color[0], blender_light.color[1], blender_light.color[2]]
+        
+        light['intensity'] = blender_light.energy
```

Comment:
I'm not aware of any relationship between Blender's `.energy` and real-world units. Assuming there is none, let's include a comment for future readers:

> Blender Render lamps have no real-world units, while glTF lights use candela (for punctual lights) and lux (for ambient and directional lights). For lack of a better conversion, use the unitless energy value here.

Perhaps we can revisit this in Blender 2.8.